src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/reporting/generator/BedUtilisationReportGenerator.kt

```diff
@@ -103,8 +107,10 @@ class BedUtilisationReportGenerator(
         effectiveTurnaroundDays = effectiveTurnaroundDays,
         voidDays = voidDays,
         totalBookedDays = totalBookedDays,
-        totalDaysInTheMonth = daysInMonth,
-        occupancyRate = totalBookedDays.toDouble() / daysInMonth,
+        bedspaceStartDate = this.createdAt.toLocalDate(),
+        bedspaceEndDate = this.endDate,
+        bedspaceOnlineDays,
```

Comment:
would be useful to explicitly use 'bedspaceOnlineDays' attributes and assign the value into the constructor like`bedspaceOnlineDays = bedspaceOnlineDays` . To avoid any issue when adding additional attribute to the bean.